echopop/workflow/feat_krill.py

```diff
@@ -0,0 +1,141 @@
+
+# ==================================================================================================
+# ==================================================================================================
+# DEFINE PARAMETERS
+# ==================================================================================================
+
+MODEL_PARAMETERS = {
+    "number_density": {"value": 500., "min": 10., "max": 10000., "vary": True},
+    "theta_mean": {"value": 10., "min": 0., "max": 90., "vary": True},
+    "theta_sd": {"value": 20., "min": 0., "max": 90., "vary": False},
+    "length_mean": {"value": 0.030, "min": 0.008, "max": 0.040, "vary": True},
+    "length_sd_norm": {"value": 0.15, "min": 0.05, "max": 0.15, "vary": False},
+    "g": {"value": 1.015, "min": 1.015, "max": 1.060, "vary": False},
+    "h": {"value": 1.020, "min": 1.015, "max": 1.060, "vary":False},
+    "radius_of_curvature_ratio": {"value": 3.0, "min": 0.5, "max": 100.0, "vary": False},
+    "length_radius_ratio": {"value": 18.2, "min": 14.0, "max": 20.0, "vary": False},
+}
+
+# Model-specific settings, including distributions
+MODEL_SETTINGS = {
+    "type": "pcdwba",
+    "taper_order": 10.,
+    "frequency_interval": 2000.,
+    "n_integration": 50,
+    "n_wavelength": 10,
+    "orientation_distribution": {"family": "gaussian", "bins": 60},
+    "length_distribution": {"family": "gaussian", "bins": 100},
+}
+
+ENVIRONMENT = {
+    "sound_speed_sw": 1500.0,
+    "density_sw": 1026.9,    
+}
+
+SIMULATION_SETTINGS = {
+    "environment": ENVIRONMENT,
+    "monte_carlo": True,
+    "mc_realizations": 2,
+    "scale_parameters": True, 
+    "minimum_frequency_count": 2,
+    "reference_frequency": 120e3,
+}
+
+OPTIMIZATION_KWARGS = {
+    "max_nfev": 1000,
+    "method": "least_squares",
+    "loss": "huber",
+    "xtol": 1e-8,
+    "ftol": 1e-8,
+    "gtol": 1e-8,
+    "diff_step": 1e-7,
+    "verbose": 2,
+    "restart_strategy": {
+        "max_attempts": 3,
+        "Q_threshold": 3.,
+        "scale": 0.10
+    },
+    "burnin": {
+        "method": "nelder",
+        "max_nfev": 100,
+        "tol": 1e-3,
+    }
+}
+
+def run_krill_inversion_workflow():
+# ==================================================================================================
+# ==================================================================================================
+# FORMAT SCATTERING MODEL PARAMETERS
+# ==================================================================================================
+
+    scattering_parameters = InvParameters(MODEL_PARAMETERS)
+
+# ==================================================================================================
+# ==================================================================================================
+# INITIALIZE
+# ==================================================================================================
+
+    INVERSION = InversionMatrix(sv_data_sub, SIMULATION_SETTINGS)
```

Comment:
I would reserve all cap for constants.
```suggestion
    invert_krill = InversionMatrix(sv_data_sub, SIMULATION_SETTINGS)
```